src/coreclr/utilcode/debug.cpp

```diff
@@ -158,11 +159,14 @@ VOID LogAssert(
     // may not be a string literal (particularly for formatt-able asserts).
     STRESS_LOG2(LF_ASSERT, LL_ALWAYS, "ASSERT:%s:%d\n", szFile, iLine);
 
-    SYSTEMTIME st;
-#ifndef TARGET_UNIX
-    GetLocalTime(&st);
+    struct timespec ts;
+    int ret = timespec_get(&ts, TIME_UTC);
```

Comment:
This method is debug only. This should be the simplest code that works cross platform.